packages/commonwealth/client/scripts/views/components/proposals/VotingActionCard.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react';
+import { CWIcon } from '../component_kit/cw_icons/cw_icon';
+import { CWText } from '../component_kit/cw_text';
+import './VotingActionCard.scss';
+import VotingResultView, { VoteOption } from './VotingResultView';
+
+interface VotingOption {
+  label: string;
+  value: string;
+  voteCount: number | string;
+}
+
+interface VotingActionCardProps {
+  options: VotingOption[];
+  proposalTitle?: string;
+  timeRemaining?: string;
+  canVote?: boolean;
+  hasVoted?: boolean;
+  onVote: (option: string | [string]) => void;
+  type: 'cosmos' | 'snapshot';
+  votingOption: VoteOption[];
+  toggleShowVotesDrawer: (newState: boolean) => void;
+  governanceUrl?: string;
+  defaultVotingOption?: string | undefined;
+}
+
+const VotingActionCard: React.FC<VotingActionCardProps> = ({
```

Comment:
please be consistent, you didn't use React FC in other components so why using it here? Also we dont use it in the codebase.